Xamarin.Essentials/DeviceDisplay/DeviceDisplay.android.cs

```diff
@@ -32,7 +33,10 @@ static bool PlatformKeepScreenOn
 
         static DisplayInfo GetMainDisplayInfo()
         {
-            var displayMetrics = Platform.AppContext.Resources?.DisplayMetrics;
+            var displayMetrics = new DisplayMetrics();
```

Comment:
It might be better to wrap this in a `using` to dispose of the object sooner. Not sure how big this object is.

Reply:
Fixed up :) let me know what you think.